WordPress/Classes/Services/SignupService.swift

```diff
@@ -1,84 +1,74 @@
 import Foundation
 import CocoaLumberjack
 import WordPressShared
+import WordPressKit
 
 
 
-typealias SignupSocialSuccessBlock = (_ newAccount: Bool) -> Void
-typealias SignupFailureBlock = (_ error: Error?) -> Void
-
-
-/// SignupService is responsible for creating a new WPCom user and blog.
-///
-open class SignupService: LocalCoreDataService {
+// MARK: - SignupService: Responsible for creating a new WPCom user and blog.
```

Comment:
This one falls, a bit, in a gray area. Reason why i've used `MARK` (since ever!) is because in a single file you may have more than one class / structure definition.

Specially lately: we use *extensions* quite a lot, to declare protocol conformances, and so on. And using `MARK` is quite useful, because it's *VERY* easy to pinpoint an extension / nested type within Xcode's dropdown:

File: `NotificationRange.swift`

<img width="222" alt="screen shot 2018-04-23 at 2 54 50 pm" src="https://user-images.githubusercontent.com/1195260/39144151-5b476e12-4706-11e8-86f4-2b511870d527.png">

--

I think we need the whole team to settle on a documentation standard, because at this precise moment... we've got different styles, and no solid guidelines, i'm afraid!

Reply:
I understand what the `MARK` does. If you think it's useful, _adding_ one is fine, but I don't understand why we'd remove the existing documenting comment which can be used elsewhere in the codebase to understand what a type or method does. Looks like this:

<img width="519" alt="screen shot 2018-04-23 at 1 02 38 pm" src="https://user-images.githubusercontent.com/517257/39147457-dfd8836a-46f6-11e8-8cc2-3aaefb237c2b.png">

`MARK` makes browsing the code easier, but the removed comment makes _understanding_ the code easier.

Reply:
Ahhh i've **CHANGED** a comment, not added a new one. You're right, reverting right away, thanks for raising this!!